classes/assets/CccReducer.php

```diff
@@ -58,16 +58,21 @@ public function reduceCss($cssFileList)
         $version = Configuration::get('PS_CCCCSS_VERSION');
         $cccFilename = 'theme-' . $this->getFileNameIdentifierFromList($files) . $version . '.css';
         $destinationPath = $this->cacheDir . $cccFilename;
+        $destinationUri = $this->getFQDN() . $this->getUriFromPath($destinationPath);
 
         if (!$this->filesystem->exists($destinationPath)) {
             CssMinifier::minify($files, $destinationPath);
         }
+        if (Tools::hasMediaServer()) {
```

Comment:
Because getFQDN function do mysql requests.

```suggestion
        if (!Tools::hasMediaServer()) {
            $destinationUri = $this->getFQDN() . $this->getUriFromPath($destinationPath);
        } else {
            $relativePath = _THEMES_DIR_ . _THEME_NAME_ . '/assets/cache/' . $cccFilename;
            $destinationUri = Tools::getCurrentUrlProtocolPrefix() . Tools::getMediaServer($relativePath) . $relativePath;
        }
```

Reply:
Fixed